pkg/sentry/fsimpl/fuse/connection_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fuse
+
+import (
+	"testing"
+
+	"gvisor.dev/gvisor/pkg/sentry/kernel"
+)
+
+// TestConnectionInitBlock tests if initialization
+// correctly blocks and unblocks the connection.
+// Since it's unfeasible to test kernelTask.Block() in unit test,
+// the code in Call() and CallAsync() are not tested here.
+func TestConnectionInitBlock(t *testing.T) {
```

Comment:
I don't know if this is the best way to test this. The behavior you're trying to test lends itself well to being tested with syscall integration tests. The syscall test framework should be the way we test these behavior modes - the only place in fuse where we have go tests is in `dev_test.go` because it tests internals of the communication layer that can't be tested properly using syscall tests,

This test from what I can see just checks if `SetInitialized` closes a specific channel. Its not clear why thats useful here. I'd create a TODO to test this behavior once the test suite is merged (after this) and then create a separate PR to test that.

Reply:
I was already originally planning to test this in the FUSE_INIT test PR

Reply:
Oh what is this test testing then? I'd just remove it and just keep the dev_test.go be the only go tests for fuse

Reply:
I tried to remove this and gave up since @avagin gave me advice on how to fix the BUILD file